### PR TITLE
chore: upgrade rusty_v8 to 0.43.1

### PR DESCRIPTION
**Pull request by bartlomieju**

### CODE REVIEW

**Review by cjihrig — DISMISSED** [review]
LGTM if the CI passes

**Comment by lucacasonato** [comment]
The WPT failures are not flakes. You need to update expectations.

**Comment by bartlomieju** [comment]
Don't merge, still failing on M1

**Review by lucacasonato — APPROVED** [review]
LGTM

**Pull request merged**
